src/drawers/lineDrawer.ts

```diff
@@ -12,25 +12,12 @@ export module Drawers {
       this._pathSelection.datum(data);
     }
 
-    public setup(area: D3.Selection) {
-      this._pathSelection = area.append("path")
-                               .classed(Line.LINE_CLASS, true)
-                               .style({
-                                 "fill": "none",
-                                 "vector-effect": "non-scaling-stroke"
-                               });
-      super.setup(area);
-    }
-
-    private _createLine(xFunction: AppliedProjector, yFunction: AppliedProjector, definedFunction: AppliedProjector) {
-      if (!definedFunction) {
-        definedFunction = (d, i) => true;
-      }
-
-      return d3.svg.line()
-                   .x(xFunction)
-                   .y(yFunction)
-                   .defined(definedFunction);
+    public setup(line: D3.Selection) {
+      this._pathSelection = line.append("path")
+                                .classed(Line.PATH_CLASS, true)
+                                .style("fill", "none")
+                                .style("vector-effect", "non-scaling-stroke");
```

Comment:
Let's not have the `vector-effect` any more, since `non-scaling-stroke` doesn't work on IE. I'm not actually sure why the property was initially added.